### DISCUSSION TITLE
Expo go ignore nfc manager

### DISCUSSION THREAD

**Discussion by KatjaKirchhofer** [expo/expo#15392] [Archived: Questions]
Hi there, I´m starting to develop an app with expo where we have to use nfc. As far as I know it is not provided in expo go. But I would like to use expo go to develop the rest of the app. Is there a way to let expo go ignore the files wit nfc manager?

**Comment by EvanBacon** [comment]
Wrap the require in a try/catch. Can also check for the existence of the native module.
